src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -61,6 +61,7 @@ import { InputTypeKind } from "../type/inputTypeKind.js";
 import { Usage } from "../type/usage.js";
 import { logger } from "./logger.js";
 import { DpgContext } from "@azure-tools/typespec-client-generator-core";
+import { capitalize } from "./utils.js";
```

Comment:
Could you use prettier to format this file?

Reply:
It is already formatted. We have add prettier check in CI, if it is not formatted, CI will fail.